core-implementation/core.ts

```diff
@@ -2,13 +2,19 @@ import grpc from "@grpc/grpc-js";
 import { addService, loadProtoService } from "../proto.js";
 import { hello } from "./hello/hello.js";
 import { recordTransaction } from "./record_transaction/record_transaction.js";
+// Added (dtd35)
```

Comment:
you dont need to attribute specific lines of code, git will do that for you.
https://git-scm.com/docs/git-blame